lib/internal/test_runner/test.js

```diff
@@ -153,20 +178,6 @@ class TestContext {
     this.#test.todo(message);
   }
 
-  test(name, options, fn) {
```

Comment:
why not keep as prototype methods?

Reply:
for some reason it did not work for me adding skip/todo/etc to the test function